### PR TITLE
Decompress outputs into `outputfolder`

### PR DESCRIPTION
**Pull request by dachengx**
1. Add bash script `alea/submitters/separate.sh` which decompresses the output of each `combine.sh` job locally.
2. Add function `_add_separate_job` which calls with `run_on_submit_node = True` to run the decompression locally.
3. Replace most `.format` to f-string (minor change)

Close: https://github.com/XENONnT/alea/issues/175

Reference: https://pegasus.isi.edu/documentation/index.html

### CODE REVIEW

**Comment by dachengx** [comment]
But the `separate.sh` job is pending here forever:
```
(XENONnT_development) xudc@ap23:~$ pegasus-status -l /scratch/xudc/workflows/runs/dx_cevns-discovery_power-202409050505STAT  IN_STATE  JOB
Run      22:15  alea_workflow-0 ( /scratch/xudc/workflows/runs/dx_cevns-discovery_power-202409050505 )
Idle     12:28   ┗━separate_ID0000002
Summary: 2 Condor jobs total (I:1 R:1)

UNRDY READY   PRE  IN_Q  POST  DONE  FAIL %DONE STATE   DAGNAME
    0     0     0     1     0    11     0  91.7 Running *alea_workflow-0.dag
```

File: alea/submitters/htcondor.py [hunk 1]
```diff
@@ -487,6 +503,26 @@ def _add_combine_job(self, combine_i):
...
         return combine_job
 
+    def _add_separate_job(self, combine_i):
+        """Add a separate job to the workflow."""
+        logger.info(f"Adding separate job {combine_i} to the workflow")
+        separate_name = "separate"
+        separate_job = self._initialize_job(
+            name=separate_name,
+            cores=self.request_cpus,
+            memory=self.request_memory * 2,
```
Comment (FaroutYLq) [inline alea/submitters/htcondor.py:513]: What is the motivation for double here?

Comment (dachengx) [reply alea/submitters/htcondor.py:513]: https://github.com/XENONnT/alea/blob/335ff8b153789f201f4afc50685ffe4fbeeba733/alea/submitters/htcondor.py#L476

Comment (dachengx) [reply alea/submitters/htcondor.py:513]: Copied from you.

Comment (FaroutYLq) [reply alea/submitters/htcondor.py:513]: OK I don't think this is a problem. Just curious

File: alea/submitters/htcondor.py [hunk 2]
```diff
@@ -487,6 +503,26 @@ def _add_combine_job(self, combine_i):
 
         return combine_job
 
+    def _add_separate_job(self, combine_i):
+        """Add a separate job to the workflow."""
+        logger.info(f"Adding separate job {combine_i} to the workflow")
+        separate_name = "separate"
+        separate_job = self._initialize_job(
+            name=separate_name,
+            cores=self.request_cpus,
```
[the remark this answers is not in the document]
Comment (dachengx) [reply alea/submitters/htcondor.py]: Done in https://github.com/XENONnT/alea/pull/199/commits/9ef04119652b8d9f7c5da311355ec1764eb174b2

**Comment by dachengx** [comment]
Key change: https://github.com/XENONnT/alea/pull/199/commits/2c37dc49dcd7669507a340a925d5b26a7f1e16f9#diff-165863f38a1128d2c2d5cc40fd983794f3619097473c9c49f9679adbad1177d0L507

**Pull request merged**